main.py

```diff
@@ -60,8 +57,14 @@ def get_edit(request: Request, item_id: int, db: Session = Depends(get_db)):
 
 
 @app.put("/edit/{item_id}", response_class=HTMLResponse)
-def put_edit(request: Request, item_id: int, content: str = Form(...), db: Session = Depends(get_db)):
-    todo = update_todo(db, item_id, content)
```

Comment:
It might be helpful to add a comment explaining the purpose of the 'notes' field for future developers.